corehq/apps/reminders/views.py

```diff
@@ -1054,7 +1064,9 @@ def broadcast_form(self):
             'location_ids': ','.join(broadcast.location_ids),
             'include_child_locations': broadcast.include_child_locations,
         }
-        return BroadcastForm(initial=initial, **self.form_kwargs)
+        if toggles.EWS_BROADCAST_BY_ROLE.enabled(self.domain):
```

Comment:
I'm not thrilled about introducing this kind of thing into this view, but unfortunately it's the only way to support this one-off feature